ios/Classes/CountlyFlutterPlugin.m

```diff
@@ -1665,6 +1665,15 @@ - (void)populateConfig:(NSDictionary *)_config {
             config.indirectAttribution = attributionValues;
         }
 
+        NSNumber *visibilityTracking = _config[@"visibilityTracking"];
+        if (visibilityTracking) {
+            config.experimental.enableVisibiltyTracking = [visibilityTracking boolValue];
```

Comment:
typo here *enableVisibilityTracking

Reply:
but this typo is not fixable here :)
It was already released in iOS

Reply:
ohhh okey :/

Reply:
:D